packages/quantic/force-app/main/default/lwc/quanticRefineToggle/quanticRefineToggle.html

```diff
@@ -0,0 +1,41 @@
+<template>
+  <button class="slds-button slds-button_neutral refine-button slds-is-relative" onclick={openModal}
```

Comment:
Instead of having attributes to customize and enable the button text and icon, why don't you define a slot for the button content instead? This way, anyone could customize the button as needed. We might want to add a property to allow customizing the button CSS classes though.

Reply:
I tried to follow the same logic we currently  using with the Quickview button but having a slot for the button content is indeed a better idea. I'll try to implement this logic 👌